index.html

```diff
@@ -116,10 +116,11 @@
 
         // Spec URLs
         ariaSpecURLs: {
-          "ED": "",
-          "WD" : "",
-          "CR": "",
-          "REC": ""
+          "ED": "https://w3c.github.io/aria/",
+          "FPWD": "https://w3c.github.io/aria/",
+          "WD": "https://w3c.github.io/aria/",
+          "CR": "https://w3c.github.io/aria/",
+          "REC": "https://w3c.github.io/aria/",
```

Comment:
Should this be all pointing to the editor's draft? shouldn't they mostly be pointing to 1.3 first public working draft?

Reply:
I basically meant to ask the same question but did so on the other side of the diff.